samples/MvcSubAreaSample.Web/Areas/Restaurant/Areas/Hours/Controllers/HomeController.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace MvcSubAreaSample.Web.Areas.Restaurant.Hours.Controllers
```

Comment:
`namespace MvcSubAreaSample`? Just so that people don't assume the namespace has to do anything with the subarea discovery.